docs/examples/.env.staging.example

```
@@ -42,10 +41,15 @@ FIRST_SUPERUSER_EMAIL=admin@yourdomain.com
-# MINIO (Object Storage)
+# RUSTFS (Object Storage — TrueNAS S3)
 # ============================================
-MINIO_ROOT_USER=minioadmin
-MINIO_ROOT_PASSWORD=CHANGE-ME-min-8-chars
+RUSTFS_HOSTNAME=s3-api.evillab.dev
+RUSTFS_PORT=443
+RUSTFS_USE_HTTPS=true
+RUSTFS_ACCESS_KEY=your-rustfs-access-key
+RUSTFS_SECRET_KEY=your-rustfs-secret-key
+RUSTFS_DEFAULT_BUCKET=fallout-shelter
+RUSTFS_PUBLIC_URL=https://s3-api.evillab.dev
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RUSTFS_PUBLIC_URL key should go before the RUSTFS_SECRET_KEY key

(UnorderedKey)